src/main/scala/eventstore/tcp/EventStoreProtoFormats.scala

```diff
@@ -120,7 +115,7 @@ trait EventStoreProtoFormats extends DefaultProtoFormats with DefaultFormats {
           data = Content(byteString(x.getData), ContentType(x.getDataContentType)),
           metadata = Content(byteString(x.getMetadata), ContentType(x.getMetadataContentType))
         ),
-        created = option(x.hasCreatedEpoch, new DateTime(x.getCreatedEpoch))
+        created = option(x.hasCreatedEpoch, ZonedDateTime.ofInstant(Instant.ofEpochMilli(x.getCreatedEpoch), ZoneOffset.UTC))
```

Comment:
why change to strict `UTC` ?

Reply:
`ZoneDateTime.ofInstant` requires a zone. `Instant.ofEpochMillis` is documented as follows:
```
* Obtains an instance of {@code Instant} using milliseconds from the
* epoch of 1970-01-01T00:00:00Z.
```
I used `ZoneOffset.UTC` given the above.

Reply:
Perhaps rename `created` to `createdUTC` as this is a breaking change anyway. Alternative is:

```scala
created = option(x.hasCreatedEpoch, ZonedDateTime.ofInstant(Instant.ofEpochMilli(x.getCreatedEpoch), ZoneId.systemDefault))
```

I would prefer the consistency that UTC provides.

Reply:
IMHO in apps that you want consistency - you set global system timezone to UTC rather than override it somewhere

Reply:
@t3hnar I updated the PR to use same behavior as  `new DateTime(x.getCreatedEpoch)` that uses default time zone.